DEPRECATED.rst

```diff
@@ -75,6 +75,11 @@ Bio.Index
 ---------
 Deprecated in release 1.75, was not used anywhere in Biopython.
 
+Bio.Crystal
+-----------
+Declared obsolete in release 1.75, there is no implementation of the classes
```

Comment:
How about just:

Declared obsolete in release 1.75, PDB NDB files can be opened with Bio.PDB.

Reply:
Done.